webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -2,25 +2,54 @@ import React from 'react';
 import { IntlProvider } from 'react-intl';
 import PropTypes from 'prop-types';
 import { translate as __ } from 'foremanReact/common/I18n';
-import { Button } from 'patternfly-react';
+import { Button, Icon } from 'patternfly-react';
 import { INSIGHTS_SYNC_PAGE_TITLE } from './InsightsCloudSyncConstants';
 
-const InsightsCloudSync = ({ syncInsights }) => {
+const InsightsCloudSync = ({ data: { settingsUrl }, syncInsights }) => {
   document.title = INSIGHTS_SYNC_PAGE_TITLE;
   return (
     <IntlProvider locale={navigator.language}>
       <div className="insights-cloud-sync">
         <h1>{__('Red Hat Insights Sync')}</h1>
-        <Button bsStyle="primary" onClick={syncInsights}>
-          {__('Sync now')}
-        </Button>
+        <p>
+          {__(`Insights synchronization process is used to provide Insights
+           recommendations output for hosts managed here`)}
+        </p>
+        <p>
+          {__(`1. Obtain an RHSM API token: `)}
+          <a
+            href="https://access.redhat.com/management/api"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            access.redhat.com <Icon name="external-link" size="xs" />
```

Comment:
Although the icons LGTM:
![6-6_insights_welcome](https://user-images.githubusercontent.com/26363699/87244458-31c57380-c446-11ea-9e3a-8443e5555201.png)

There is the following error:
```
Warning: Failed prop type: Invalid prop `size` of value `xs` supplied to `FontAwesome`, expected one of ["lg","2x","3x","4x","5x"].
    in FontAwesome (created by Icon)
    in Icon (created by InsightsCloudSync)
    in a (created by InsightsCloudSync)
    in p (created by InsightsCloudSync)
    in div (created by InsightsCloudSync)
    in IntlProvider (created by InsightsCloudSync)
    in InsightsCloudSync (created by Connect(InsightsCloudSync))
    in Connect(InsightsCloudSync) (created by I18nProviderWrapper(Connect(InsightsCloudSync)))
    in IntlProvider (created by I18nProviderWrapper(Connect(InsightsCloudSync)))
    in I18nProviderWrapper(Connect(InsightsCloudSync)) (created by StoreProvider(I18nProviderWrapper(Connect(InsightsCloudSync))))
    in Provider (created by StoreProvider(I18nProviderWrapper(Connect(InsightsCloudSync))))
    in StoreProvider(I18nProviderWrapper(Connect(InsightsCloudSync))) (created by DataProvider(StoreProvider(I18nProviderWrapper(Connect(InsightsCloudSync)))))
    in DataProvider(StoreProvider(I18nProviderWrapper(Connect(InsightsCloudSync))))
```